newrelic_lambda_cli/integrations.py

```diff
@@ -428,8 +460,12 @@ def remove_integration_role(input):
     click.echo(
         "Waiting for stack deletion to complete, this may take a minute... ", nl=False
     )
-    client.get_waiter("stack_delete_complete").wait(StackName=stack_name)
-    success("Done")
+    try:
+        client.get_waiter("stack_delete_complete").wait(StackName=stack_name)
+    except botocore.exceptions.WaiterError as e:
+        failure(e.last_response["Status"]["StatusReason"])
```

Comment:
Same control flow change here.

Reply:
This is also not handled. So won't re-raise here, either.

Reply:
I'm pretty confused by your reasoning. You're saying the exception wasn't caught, which means it used to terminate the process. And now we catch it, print a message, and move on as if it had succeeded. Can you say a bit more about why that's better? I'm not getting it.

Reply:
These two cases are removing resources. In both cases, the waiter error being  raised would just result in an unhandled exception resulting in the traceback being dumped to the terminal. However, catching the exception and displaying the status reason, in my view, is an improved user experience.

Additionally, we ask for confirmation by default when removing resources. And the resource removal is isolated in that a failure doesn't cause problems for a subsequent removal. As such, the control flow isn't "broken" in these cases. Instead, it is handled more gracefully and by default the user can interrupt the process.

Reply:
Ah, that makes more sense.  Thanks for the explanation!